docs/docs/test-runner/commands.md

```diff
@@ -332,6 +332,84 @@ it('natively holds and then releases a key', async () => {
 
 </details>
 
+### Select option
+
+The `selectOption` command allows you to select an option in a `<select>` tag by either a single value, a label string or as multiple values, depending on the specific implementations of the browser laucher you're using. Once selected, the native `change` and `input` events are dispatched automatically. The `selectOption` command needs the CSS selector to locate the `<select>` by and the value, label, or values to select. The function is async and should be awaited.
+
+The three major launchers all have different support for selecting options, and the `selectOption` takes this into account.Attempting to select an option via a method that is not implemented in the provided launcher will produce errors. Each supported launcher's option selection support is documented below.
+
+- Playwright
+  - Supports selecting options by `label`, `value`, and `multiple values`
+- Puppeteer
+  - Supports selection options by `value`, and `multiple values` only. Selecting options by label is not implemented
+- WebDriver
+  - [No support for selecting options in JS](https://www.selenium.dev/documentation/webdriver/elements/select_lists/)
+
+<details>
+<summary>View example</summary>
+
+```js
+import { selectOption } from '@web/test-runner-commands';
+
+// Assuming the document.body has a select like:
+// <select id="testSelect">
+//   <option value="first">first option</option>
+//   <option value="second">second option</option>
+//   <option value="third">third option</option>
+// </select>
+
+it('natively selects an option by value', async () => {
+  const valueToSelect = 'first';
+  const select = document.querySelector('#testSelect');
+
+  await selectOption({ selector: '#testSelect', value: valueToSelect });
```

Comment:
```suggestion
  await selectOption({ selector: select value: valueToSelect });
```

that should work right?

Reply:
yep, it should.

as far as I can tell, its any CSS selector. do you think I should include another test using another selector? I thought about it, but wasnt sure how much value that would provide since there's always several ways to target some HTML element, should we just trust the locator mechanism from the drivers, or should we test them explicitly?

Reply:
yeah I think we can trust that it works 👍
in the docs, however, it's "strange" as you already have an instance via `querySelector('#testSelect');` and in the following line you "query it again"...

Reply:
oh, the query is for the `expect` so that I can compare the values. In Playwright and Puppeteer both, the selection commands I'm using only return the "values" that were selected. So I `querySelector` my templated `select` element so that I can make sure that using the `selectOption` command actually changed the `value`. I don't see how I would test that without querying the select for the expect.

If you have a suggestion about that, please let me know!

Reply:
if I'm reading right, your suggestion is about passing the `select` instance into the `selectOption` command, but the select option commands from  Playwright/Puppeteer only accept CSS selectors, not DOM instances.

https://playwright.dev/docs/api/class-page#page-locator
https://pptr.dev/api/puppeteer.page.select#parameters

Reply:
yeah I was assuming the playwright API...  (which IMHO is nice)
```js
selectEl.selectOption(value);
```

in that case, you could pass `selectEl` directly or do a locator before (if it is a string)

but the puppeteer API is not as flexible
```js
page.select(selector, value); // where selector needs to be a string
```

so I guess the lowest common denominator means the "worse" API... but it's at least consistent 🤷‍♂️
otherwise, we would need 2 separate ways for playwright and puppeteer... worth it or not?

what do you think?

Reply:
hrm, good question. I don't really think its worth it. I think in common practice for testing, the selectors passed are probably going to be hardcoded a lot of the time. I know i hardcode test selectors a LOT :). So in that case, there doesnt need to be a querySelector for the `select` necessarily because selecting the option is probably just a middle step in some other test where you're testing some other UI functionality.

Therefore, I think I'd vote for the consistent API even if its "worse".

Reply:
fair enough 👍